README.md

```diff
@@ -45,51 +55,51 @@ Once imported, you can open a connection to your WebSocket server.
 
 ```swift
 
-socket = WebSocket(url: NSURL(string: "ws://localhost:8080/")!)
+socket = WebSocket(url: URL(string: "ws://localhost:8080/")!)
 socket.connect()
 ```
 Now you can subscribe e.g to all of the websocket events:
 
 ```swift
-socket.rx.response.subscribeNext { (response: WebSocketEvent) in
+socket.rx.response.subscribe(onNext: { (response: WebSocketEvent) in
 	switch response {
-	case .Connected:
+	case .connected:
 		print("Connected")
-	case .Disconnected(let error):
+	case .disconnected(let error):
 		print("Disconnected with optional error : \(error)")
-	case .Message(let msg):
+	case .message(let msg):
 		print("Message : \(msg)")
-	case .Data(_):
+	case .data(_):
 		print("Data")
-	case .Pong:
+	case .pong:
 		print("Pong")
   	}
-}.addDisposableTo(disposeBag)
+}).disposed(by self.disposeBag)
```

Comment:
Let's get rid of `self` here

Reply:
Just followed the previous example (v0.7), but yeah I agree on this one!